flang/lib/Lower/Bridge.cpp

```diff
@@ -107,6 +107,65 @@ struct IncrementLoopInfo {
   mlir::Block *exitBlock = nullptr;   // loop exit target block
 };
 
+/// Helper class to generate the runtime type info global data. This data
+/// is required to described derived type to the runtime so that it can operate
```

Comment:
Should read something like "required to describe the derived type" rather than "required to described type".